### ISSUE TITLE
Can Create a tech profile and Sign in.... Why ?

### ISSUE THREAD

**Issue by paulj199** [MicrosoftDocs/feedback#1233]
Issue is im trying to sign in and create a tech profile, nothing is happenning...

Get to https://www.microsoft.com/en-us/learning/training.aspx using my microsoft account...

![image](https://user-images.githubusercontent.com/44790798/53794476-f6e52e80-3f27-11e9-831b-1d19f0078e4c.png)

Click the start learning button taken to https://docs.microsoft.com/en-gb/learn/  but now asked to sign in..

![image](https://user-images.githubusercontent.com/44790798/53794556-1b410b00-3f28-11e9-97b5-3b4ea933e61b.png)

click sign in, taken to https://techprofile.microsoft.com/en-GB/register?replyUrl=https://docs.microsoft.com/en-gb/learn/&  where i fill out, hit register and nothing happens...

![image](https://user-images.githubusercontent.com/44790798/53794686-63f8c400-3f28-11e9-9763-33330d666b5c.png)

**Expected behavior**
I'k like to create a tech profile, and enter the site, access courses and track progress.  I do have a profile with the same email address on  https://academy.microsoft.com/en-us/dashboard  so i dont know if that needs to be linked at all...

**Desktop (please complete the following information):**
 - Windows 10
 - firefox, but have tried it in private window and on chrome and on safari on my mac, same issue

**Comment by simondavidhill** [comment]
I have exactly the same issue, except in my browser (Chrome) I get a message in red after I click the 'Register' button, which states: "Something went wrong. please try again later."

**Comment by simondavidhill** [comment]
Try setting country to US as per this comment: https://github.com/MicrosoftDocs/feedback/issues/1234#issuecomment-469759236

**Comment by stillface** [comment]
No Works even I selected US. Same message: "Something went wrong. please try again later."

**Comment by paulj199** [comment]
Yep..   Same here selecting US doesnt fix, i get the "something wrong" message in chrome and on Safari but dont get it on firefox.

**Comment by richardiriwn** [comment]
Same problem, have been going around in circles on your new partner site for a couple of weeks. - no message, just does nothing Using Edge on Windows 10

**Comment by saldana** [comment]
@jialge please take a look and review the feedback from this customer. Thanks.

**Comment by markclanfalconer** [comment]
I am getting the same

Examining traffic I see

`
Request URL: https://docs.microsoft.com/api/profiles
Request method: OPTIONS
Remote address: 2.20.37.130:443
Status code: 403 Forbidden
Version: HTTP/2.0
Referrer Policy: no-referrer-when-downgrade

REQUEST HEADERS
Accept: text/html,application/xhtml+xm…ml;q=0.9,image/webp,*/*;q=0.8
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.5
Access-Control-Request-Headers: content-type
Access-Control-Request-Method: POST
Connection: keep-alive
DNT: 1
Host: docs.microsoft.com
Origin: https://techprofile.microsoft.com
Referer: https://techprofile.microsoft.…&activate-azure-sandbox=true&&
TE: Trailers
User-Agent: Mozilla/5.0 (Windows NT 6.1; W…) Gecko/20100101 Firefox/65.0
`

**Comment by saeedlabs** [comment]
> with Firefox is worked for me without check the "I agree" box......

can confirm this worked for me too.

Leaving country blank did not work. Tested on both edge and chrome.

**Comment by saldana** [comment]
@dastaffo Adding you here as well to take a look.
